src-opam/dockerfile_windows.ml

```diff
@@ -47,18 +54,34 @@ let install_visual_studio_build_tools ?(vs_version="16") ?(split=false) componen
   @@ add ~src:["https://aka.ms/vs/" ^ vs_version ^ "/release/vs_buildtools.exe"] ~dst:{|C:\TEMP\vs_buildtools.exe|} ()
   @@ install
 
-let append_path paths =
+let prepend_path paths =
   let paths = String.concat ";" paths in
   run {|for /f "tokens=1,2,*" %%a in ('reg query "HKLM\SYSTEM\CurrentControlSet\Control\Session Manager\Environment" /V Path ^| findstr /r "^[^H]"') do `
-       reg add "HKLM\SYSTEM\CurrentControlSet\Control\Session Manager\Environment" /V Path /t REG_EXPAND_SZ /f /d "%%c;%s"|} paths
+        reg add "HKLM\SYSTEM\CurrentControlSet\Control\Session Manager\Environment" /V Path /t REG_EXPAND_SZ /f /d "%s;%%c"|} paths
+
+type windows_port = [
+  | `Cygwin
```

Comment:
`` `Cygwin`` is a Unix distribution, it shouldn't be here. I'm anticipating that we'll make it work by providing an equivalent OS base image (say to the Ubuntu ones) which just happens itself to be based on Windows.